docs/features.md

```diff
@@ -33,6 +33,12 @@ not running will not be updated when karydia starts.
 
 The network policy is expected to be found under `.data.policy` in the configmap.
 
+The current network policy called `karydia-default-network-policy` has two security measures:
+1. block access to host network (AWS only)
+2. block access to Meta Data Services (AWS, GCP, Azure, Alibaba Cloud)
+
+Note: The network policy is still quite open. It uses a blacklisting approach and does not block Internet access (Egress) which is necessary for the protection of malware.
```

Comment:
I think we shouldn't mention `which is necessary for the protection of malware` at all. 😈

Reply:
@SAPJonasZengerle 
In my opinion, it is a security hint that can be mentioned. Users of karydia should be aware of the fact that it is not the most secure network policy and that there are certain security limitations.